src/plugins/discover/public/application/apps/main/components/doc_table/components/pager/tool_bar_pagination.tsx

```diff
@@ -27,6 +28,11 @@ interface ToolBarPaginationProps {
   onPageSizeChange: (size: number) => void;
 }
 
+const TOOL_BAR_PAGINATION_STYLES = {
+  marginLeft: 'auto',
+  marginRight: themeLight.euiSizeL,
```

Comment:
is `L` the minimum margin size which resolves the problem? would `m` do the trick?

Reply:
Partially

![CE42D551-A7E5-42E4-B9D2-AE280E7A1464_4_5005_c](https://user-images.githubusercontent.com/39378793/140337163-a02a0ee7-9e8f-4136-8146-b4c181445701.jpeg)